### DISCUSSION TITLE
Initial commit of Dockerfile (Still needs Github compatibility)

### DISCUSSION THREAD

**Discussion by ACraig7** [DrAlzahraniProjects/csusb_fall2024_cse6550_team1#2] [General]
I pushed the initial commit of the Dockerfile. If we need more dependencies, let me know. Also, to add a Github dependencies to the Dockerfile we need to use an SSH token or a private key and as far as I can tell i do not have access to that. I think Logan would have to set that up, unless I am missing something. Let me know if you have any questions.

**Comment by itsnotmik** [comment]
I followed up with your initial commit and expanded on the functionality. The docker image can now be built + ran and will display a simple hello world statement through StreamLit.

And from my understanding, we shouldn't need to have SSH tokens since we won't need to pull from the GitHub. We can copy the files needed directly from the directory of the docker file. I do this with the app.py file and am able to run it on the docker image just fine

**Reply by ACraig7** [reply]
From my understanding the dockerfile needs to integrate github according to the syllabus. The idea behind this is so that every time you build the docker image it clones the github main branch. If we were to do this we would need SSH tokens since the repo is private.
